### PR TITLE
feat: add badge benefit to progress modal copy

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: client/src/redux/index.js [hunk 1]
```diff
@@ -235,14 +234,8 @@ export const shouldRequestDonationSelector = state => {
   // donations only appear after the user has completed ten challenges (i.e.
   // not before the 11th challenge has mounted)
   // the follwoing is an AB test for increasing the completed challenge requirement to 20
```
Comment (ieahleen) [inline client/src/redux/index.js:236]: if you are removing that AB test maybe you want to remove this comment

Comment (ahmadabdolsaheb) [inline client/src/redux/index.js:236]:
```suggestion
```

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 

---

I do wonder if it might make sense to include an example of what a "supporter profile badge" is?

**Pull request merged**